hls4ml/utils/config.py

```diff
@@ -32,12 +32,24 @@ def _get_precision_from_quantizer(quantizer):
             quantizer['class_name'] = quantizer_obj.__class__.__name__
             quantizer['config'] = quantizer_obj.get_config()
         # Some activations are just functions
-        else: 
+        else:
             quantizer['class_name'] = quantizer_obj.__name__
 
-    supported_quantizers = ['quantized_bits', 'quantized_relu', 'quantized_tanh', 'quantized_po2', 'quantized_relu_po2']
+    supported_quantizers = ['quantized_bits', 'quantized_relu', 'quantized_tanh',
+                            'quantized_sigmoid', 'quantized_po2', 'quantized_relu_po2']
     signed = True
-    if quantizer['class_name'] in supported_quantizers:
+    rnd = "AP_RND_CONV"
+    overflow = "AP_SAT"
```

Comment:
These will apply to weights as well as activations. It may not matter, since the 'rounding' for weights actually happens in Python and not HLS, but we should make sure it doesn't have any implications on the HLS results.

Reply:
I was wondering whether I should put the ap_fixed defaults (AP_TRN and AP_WRAP) or these as the default, and didn't really have convincing arguments one way or the other. The QONNX route tends to have saturation and rounding, so I chose that at the end, but it was not a strong argument.

Reply:
I think we just need to see that:
- no QKeras tests that were passing now fail
- resources are the same for some synthesized QKeras models on master and on this branch, with all else being the same

Reply:
I was wondering if we should defer this to the OutputRoundingSaturationMode optimizer. That might be more consistent with what is done in other cases. I will try that.

Reply:
I decided to go back to using the ap_fixed defaults.